tendermint/src/error.rs

```diff
@@ -208,8 +208,7 @@ define_error! {
             |_| { format_args!("subtle encoding error") },
 
         Signature
-            [ DisplayOnly<signature::Error> ]
-            |_| { format_args!("signature error") },
+            |_| { "signature error" },
```

Comment:
Similar to my other comment: should we not find a way here to bubble up the original error?

Reply:
xref https://github.com/informalsystems/tendermint-rs/pull/1046#discussion_r765322068